apps/jetpackme-support/webpack.config.js

```diff
@@ -0,0 +1,100 @@
+const path = require( 'path' );
+const FileConfig = require( '@automattic/calypso-build/webpack/file-loader' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const TranspileConfig = require( '@automattic/calypso-build/webpack/transpile' );
+const { shouldTranspileDependency } = require( '@automattic/calypso-build/webpack/util' );
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
+const ReadableJsAssetsWebpackPlugin = require( '@wordpress/readable-js-assets-webpack-plugin' );
+const autoprefixerPlugin = require( 'autoprefixer' );
+const webpack = require( 'webpack' );
+const cacheIdentifier = require( '../../build-tools/babel/babel-loader-cache-identifier' );
+const GenerateChunksMapPlugin = require( '../../build-tools/webpack/generate-chunks-map-plugin' );
+const isDevelopment = process.env.NODE_ENV !== 'production';
+const extraPath = 'fallback';
+const cachePath = path.resolve( '.cache', extraPath );
+
+/* Arguments to this function replicate webpack's so this config can be used on the command line,
+ * with individual options overridden by command line args.
+ * @see {@link https://webpack.js.org/configuration/configuration-types/#exporting-a-function}
+ * @see {@link https://webpack.js.org/api/cli/}
+ * @param   {Object}  env                           environment options
+ * @param   {string}  env.source                    plugin slugs, comma separated list
+ * @param   {Object}  argv                          options map
+ * @param   {string}  argv.entry                    entry path
+ * @returns {Object}                                webpack config
+ */
+function getWebpackConfig( env = { source: '' }, argv = {} ) {
+	env.WP = true;
+	const outputPath = path.join( __dirname, 'dist' );
+
+	const webpackConfig = getBaseWebpackConfig( env, argv );
+
+	return {
+		...webpackConfig,
+		mode: isDevelopment ? 'development' : 'production',
+		entry: {
+			'jetpackme-support': path.join( __dirname, 'index.js' ),
+		},
+		output: {
+			...webpackConfig.output,
+			path: outputPath,
+			filename: '[name].min.js', // dynamic filename
+			library: 'jetpackme-support',
```

Comment:
Looks like it also sets `libraryTarget`, which should wind up doing about the same thing. Plus I'm not sure whether the `-` is allowed.